internal/provider/baremetal/inventory.go

```diff
@@ -13,10 +17,54 @@ func (p *baremetalProvider) CleanPlatformValuesFromDBUpdates(_ map[string]interf
 	return nil
 }
 
+// GetActualSchedulableMasters returns the value of the SchedulableMasters cluster
+// property in case it was actively set through the API. Otherwise it returns
+// the expected default value depending on the number of hosts in the cluster.
+func (p *baremetalProvider) GetActualSchedulableMasters(cluster *common.Cluster) (bool, error) {
+	if cluster == nil {
+		return false, errors.New("unexpected 'nil' cluster")
+	}
+	if cluster.SchedulableMasters != nil {
+		return *cluster.SchedulableMasters, nil
+	}
+	if common.CountActiveHosts(cluster) < 5 {
```

Comment:
* try using cluster.ReadyHostCount.
* probably worth extracting '5' to a const and explain that restriction in scheduling.

Reply:
It's used in get cluster as well, so if we start the installation you probably want to get the right reply there as well. 
`ReadyHostCount` count hosts only in known state.

Reply:
so EnabledHostCount then? https://github.com/openshift/assisted-service/blob/b9ab5d3c77580ab93c29bb43cb78cc6b02d54e88/internal/common/db.go#L361

Reply:
yes that will be better

Reply:
done